docusaurus/docs/Android/03-ui/06-guides/08-adding-custom-attachments.mdx

```diff
@@ -39,6 +43,7 @@ First of all, you'll need to customize the leading content of [MessageComposerVi
             android:layout_height="32dp"
             android:padding="4dp"
             android:src="@drawable/ic_calendar"
```

Comment:
Could we add something like a note saying that the user has to add the resource, just to save some possible unnecessary frustration from their side.